src/Sulu/Component/Content/ContentTypeManager.php

```diff
@@ -60,6 +60,11 @@ public function get($alias)
         return $this->container->get($serviceId);
     }
 
+    /**
+     * @param string $alias
+     *
+     * @return bool
+     */
```

Comment:
This is defined on the interface and so should not be required to be defined here: https://github.com/sulu/sulu/blob/08a40b7de9c1b3f9a6c42edc928ee8e4ff9ea97c/src/Sulu/Component/Content/ContentTypeManagerInterface.php#L33

Reply:
Removed it and fixed the parameter name to match the one documented in the interface.